tools/topology/m4/mixercontrol.m4

```diff
@@ -31,20 +31,30 @@ define(`CONTROLMIXER_OPS',
 
 define(`N_CONTROLMIXER', `CONTROLMIXER'PIPELINE_ID`.'$1)
 
-dnl C_CONTROLMIXER(name, index, ops, max, invert, tlv, comment, KCONTROL_CHANNELS, useleds, ledsdir)
+dnl C_CONTROLMIXER(name, index, ops, max, invert, tlv, comment,
+dnl		   KCONTROL_CHANNELS, useleds, ledsdir, core)
 define(`C_CONTROLMIXER',
-`ifelse(`$#', `10',
+`ifelse(`$#', `11',
 `SectionVendorTuples."'N_CONTROLMIXER($2)`_tuples_w" {'
 `	tokens "sof_led_tokens"'
 `	tuples."word" {'
-`		SOF_TKN_MUTE_LED_USE'		$9
-`		SOF_TKN_MUTE_LED_DIRECTION'	$10
+`		SOF_TKN_MUTE_LED_USE'		$10
+`		SOF_TKN_MUTE_LED_DIRECTION'	$11
 `	}'
 `}'
 `SectionData."'N_CONTROLMIXER($2)`_data_w" {'
 `	tuples "'N_CONTROLMIXER($2)`_tuples_w"'
 `}'
 ,` ')'
+`SectionVendorTuples."'N_CONTROLMIXER($2)`_tuples_comp_w" {'
+`	tokens "sof_comp_tokens"'
+`	tuples."word" {'
+`		SOF_TKN_COMP_CORE_ID'		STR($9)
```

Comment:
is this correct? The comment shows the core is the last argument?

Reply:
@ranj063 the comment is wrong, thanks!